src/main/java/org/mockito/ArgumentCaptor.java

```diff
@@ -100,12 +84,41 @@ private ArgumentCaptor(Class<? extends T> clazz) {
      * @return null or default values
      */
     public T capture() {
-        Mockito.argThat(capturingMatcher);
+        return captureIf(Any.ANY);
```

Comment:
In the current implementation I use the `and` and `or` matcher to achieve this. I don't see the need of `isNullOrA`. I think it's a better solution